zmscitizenapi/tests/Zmscitizenapi/Controllers/Office/SourceCacheWarmupControllerTest.php

```
@@ -0,0 +1,92 @@
+
+use BO\Zmscitizenapi\Utils\ErrorMessages;
+use BO\Zmscitizenapi\Tests\ControllerTestCase;
+
+class SourceCacheWarmupControllerTest extends ControllerTestCase
+{
+    protected $classname = "\BO\Zmscitizenapi\Controllers\System\SourceCacheWarmupController";
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        \App::$source_name = 'unittest';
```

PHPStan (2.2.7)

[error] Access to an undefined static property App::$source_name.


(staticProperty.notFound)